samples/dailymotion-simple-cmdline-sample/pom.xml

```diff
@@ -37,7 +37,7 @@
         <artifactId>maven-checkstyle-plugin</artifactId>
         <version>3.0.0</version>
```

Comment:
You can upgrade this to 3.1.0. That might have slightly more up to date google checks.

Reply:
I'll try that, but will have to only run it on Java 8+

Reply:
In that case I wonder if we could simply copy and paste the newer googlechecks.xml into our own checkstyle config and stay at 3.0.0?